test/api.test.js

```diff
@@ -840,7 +840,7 @@ describe("api.js", () => {
                 "cpu_usagePercentage_q1": 0,
                 "cpu_usagePercentage_q3": 0
             }]);
-        });
+        }).timeout(10000);
```

Comment:
Why? We had this before, extending the max time here does not change anything, since the test does not really vary in its runtime. The one thing that can make a difference for catching the memory/cpu metrics (if that's still the intermittent issue) is the sleep() on line 806. cc @jdelbick